website/docs/frontend/plugin-ui/getting-started.md

```
@@ -0,0 +1,356 @@
+| **Plugin group with `.init()`** | `JobsPlugin.init({ plugins: [...] })` | Parent plugin with customizable child plugins |
+| **Conditional loading** | `...(condition ? [Plugin] : [])` | Feature-flagged or environment-gated plugins |
+
+## Step 1: Create the Plugin Directory
+
+```bash
+mkdir -p packages/plugins/my-plugin-ui/src/lib
+```
+
+Standard directory structure:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)